homeassistant/components/calendar/__init__.py

```diff
@@ -160,7 +165,7 @@ def state_attributes(self):
         }
 
     @property
-    def state(self):
+    def state(self) -> str:
         """Return the state of the calendar event."""
         if (event := self.event) is None:
             return STATE_OFF
```

Comment:
Side note: We should consider returning None here to represent unknown state to allow entities to set that state.

Reply:
I went ahead and updated this.

Reply:
Then this is a breaking change. Maybe we should do it in a separate PR?

Reply:
Oh, I see you just updated the typing. Ok.